templates/_helpers.tpl

```diff
@@ -372,13 +372,13 @@ Sets extra injector service annotations
 Sets extra injector webhook annotations
 */}}
 {{- define "injector.webhookAnnotations" -}}
-  {{- if .Values.injector.webhookAnnotations }}
+  {{- if or (((.Values.injector.webhook)).annotations) (.Values.injector.webhookAnnotations)  }}
```

Comment:
This double parenthesis pattern is new to me - it looks like it's behaving a bit like a null coalescing operator?

Reply:
Yes, you are 100% correct! I have put this in place to support the new fields without them being required. Making use of the Go text processing functionality.